server/utils/utils.ts

```diff
@@ -99,6 +100,17 @@ export const sortComponentRequestData = (dataItem: HasRepoName, compareDataItem:
   return 0
 }
 
+export const sortProductIdData = (dataItem: hasProductId, compareDataItem: hasProductId) => {
+  if (dataItem.attributes.p_id < compareDataItem.attributes.p_id) {
+    return -1
+  }
+  if (dataItem.attributes.p_id > compareDataItem.attributes.p_id) {
+    return 1
+  }
```

Comment:
for comparing strings you can use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare).